studio/components/interfaces/BillingV2/Usage/Usage.tsx

```diff
@@ -0,0 +1,166 @@
+import clsx from 'clsx'
+import { useParams } from 'common'
+import ShimmeringLoader from 'components/ui/ShimmeringLoader'
+import { useProjectsQuery } from 'data/projects/projects-query'
+import { useProjectSubscriptionQuery } from 'data/subscriptions/project-subscription-query'
+import { useProjectUsageQuery } from 'data/usage/project-usage-query'
+import Link from 'next/link'
+import { useRef, useState } from 'react'
+import { Button, IconAlertCircle, IconCheckCircle, IconLoader, Listbox, Toggle } from 'ui'
+import Activity from './Activity'
+import Bandwidth from './Bandwidth'
+import Infrastructure from './Infrastructure'
+import SizeAndCounts from './SizeAndCounts'
+import { USAGE_CATEGORIES, USAGE_STATUS } from './Usage.constants'
+import { getUsageStatus } from './Usage.utils'
+
+const Usage = () => {
+  const { ref } = useParams()
+  const [selectedProjectRef, setSelectedProjectRef] = useState<string>(ref as string)
+
+  const infrastructureRef = useRef<HTMLDivElement>(null)
+  const bandwidthRef = useRef<HTMLDivElement>(null)
+  const sizeAndCountsRef = useRef<HTMLDivElement>(null)
+  const activityRef = useRef<HTMLDivElement>(null)
+
+  const { data: projects, isLoading } = useProjectsQuery()
+  const { data: usage } = useProjectUsageQuery({ projectRef: ref })
+  const { data: subscription, isLoading: isLoadingSubscription } = useProjectSubscriptionQuery({
+    projectRef: selectedProjectRef,
+  })
+
+  const scrollTo = (id: 'infra' | 'bandwidth' | 'sizeCount' | 'activity') => {
```

Comment:
Could we use url hashes (like #bandwidth), so we could even link into the page and immediately scroll to the section?

Can also be done as a future improvement tho

Reply:
no worries! i can just add the ids to each div either way, will do so